test/postman/test.postman_collection.json

```diff
@@ -0,0 +1,2117 @@
+{
+	"info": {
+		"_postman_id": "35ab0ff9-55a1-4746-879e-f2ba1e6e0fec",
+		"name": "CI test",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
```

Comment:
Very nice initial test scenario with the alternating read (list) and write (add/edit) requests!

Reply:
Thanks for that, I tried to implement something similar to the real use cases.